crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -1435,3 +1525,173 @@ fn main() {
         false,
     );
 }
+
+#[test]
+fn test_comment_highlighting_disabled() {
+    // Test that comments are not highlighted when disabled
+    check_highlighting_with_config(
+        r#"
+// This is a regular comment
+/// This is a doc comment
+fn main() {
+    // Another comment
+    println!("Hello, world!");
+}
+"#,
+        HighlightConfig {
+            strings: true,
```

Comment:
Please use `..HL_CONFIG` for the other fields instead of specifying them explicitly.